src/main.rs

```diff
@@ -50,27 +54,23 @@ fn draw(app: &mut App, gfx: &mut Graphics, state: &mut State) {
     let width = W / 2.0;
     let scale = width / state.textures[0].width();
 
+    draw.set_blend_mode(Some(BlendMode::OVER));
```

Comment:
This set's the blend mode to `Over` for anything drawn that doesn't override the blend_mode by itself.